client/src/containers/MainContentField/style.js

```diff
@@ -0,0 +1,14 @@
+import { makeStyles } from "@material-ui/core/styles";
+
+export const useStyles = makeStyles((theme) => ({
+  root: {
+    // margin: theme.spacing(2),
```

Comment:
make sure you clean up this commented out code if not unused before merging :)

Reply:
Removed commented out code